Araali-Tools/GoShelly/cobrashelly/cmd/root.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright © 2022 Soham Arora arorasoham9@gmail.com
```

Comment:
Author? Do you really want to give your email?

Reply:
Shouldn't be there. Will remove.